.gitignore

```diff
@@ -6,3 +6,4 @@ examples/Container.php
 examples/Definitions/
 examples/preload.php
 tests/Definitions/
+.idea
```

Comment:
system-specific ignores should go to your global gitignore (https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)